alarm/AlarmManager2.js

```diff
@@ -1220,9 +1233,6 @@ module.exports = class {
 
     let userFeedback = info.info;
```

Comment:
userFeedback is a confusing name, change it to something else

Reply:
Just keep the name same with "alarm:allow". It is more friendly when iOS developer plugin API.